Makefile

```diff
@@ -100,12 +104,18 @@ gen-crds: $(OPERATOR_SDK)
 	$(OPERATOR_SDK) generate crds
 
 manifests:
-	$(GO) run hack/render-manifests.go $(NAMESPACE) $(HANDLER_IMAGE) $(PULL_POLICY) deploy/ $(MANIFESTS_DIR)
+	$(GO) run hack/render-manifests.go $(NAMESPACE) $(HANDLER_IMAGE) $(OPERATOR_IMAGE) $(PULL_POLICY) deploy/ $(MANIFESTS_DIR) handler
+operator-manifests:
```

Comment:
We need just one "manifests:" since we are going to manually render manifest for operator the handler ones are done by the operator itself.